.circleci/config.yml

```diff
@@ -136,11 +136,11 @@ jobs:
             - checkout
             - restore_cache:
                   keys:
-                      - v0.7-code_quality-{{ checksum "setup.py" }}
-                      - v0.7-code-quality
+                      - v0.7-code_quality-pip-{{ checksum "setup.py" }}
```

Comment:
without a prefix (here `pip`), and while using the current wrong `v0.7-code_quality-{{ checksum "setup.py" }}-site-packages` below, we will get `site-package` restored at this step, which is wrong, confusing and difficult to reason out when something is wrong.

(It happened at the CI for the commit `4.32.0.dev0` while I am debugging with @lhoestq )